### PR TITLE
AzureStack BackupAdmin update tests and Session Records

### PR DESCRIPTION
**Pull request by daozha**
## Description
AzureStack BackupAdmin update tests and Session Records

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/AzureStack/Admin/BackupAdmin/Backup.Admin.Tests/src/BackupLocationTests.cs [hunk 1]
```diff
@@ -103,10 +103,13 @@ public void TestUpdateBackupLocation()
 
                 var backupLocation = client.BackupLocations.Get(ResourceGroupName, "local");
 
-                backupLocation.Path = @"\\100.68.73.194\Share";
-                backupLocation.UserName = @"Administrator";
+                backupLocation.Path = @"\\su1fileserver\SU1_Infrastructure_2\BackupStore";
+                backupLocation.UserName = @"azurestack\azurestackadmin";
                 backupLocation.Password = "password";
                 backupLocation.EncryptionKeyBase64 = "YVVOa0J3S2xTamhHZ1lyRU9wQ1pKQ0xWanhjaHlkaU5ZQnNDeHRPTGFQenJKdWZsRGtYT25oYmlaa1RMVWFKeQ==";
```
Comment (deathly809) [inline src/AzureStack/Admin/BackupAdmin/Backup.Admin.Tests/src/BackupLocationTests.cs]: You did randomly generate this key correct?

**Comment by deathly809** [comment]
@daozha LGTM, just make sure all secrets and internal values have been removed.

**Review by dsgouda — APPROVED** [review]
LGTM
Please address @deathly809 's comments and we can merge

File: src/AzureStack/Admin/BackupAdmin/Backup.Admin.Tests/src/BackupLocationTests.cs (hunk 1 above)
Comment (daozha) [reply src/AzureStack/Admin/BackupAdmin/Backup.Admin.Tests/src/BackupLocationTests.cs]: You mean the encryption key? I didn't generate another random key but I think the original one in the test case should be randomly generated. Do you have any concern?

Comment (daozha) [reply src/AzureStack/Admin/BackupAdmin/Backup.Admin.Tests/src/BackupLocationTests.cs]: Anyway, regenerated a random key and replaced the existing one. I don't find any other secrets exposed, let me know if you have any other concern.

Comment (deathly809) [reply src/AzureStack/Admin/BackupAdmin/Backup.Admin.Tests/src/BackupLocationTests.cs]: Looks good to me, just wanted to make sure before we push.

**Comment by dsgouda** [comment]
@deathly809 is this ready to merge?

**Comment by deathly809** [comment]
@dsgouda Yeah.

**Pull request merged**